src/taipy/gui/gui.py

```diff
@@ -1541,6 +1546,14 @@ def broadcast(self, name: str, value: t.Any):
         """
         self.__send_ws_broadcast(name, value)
 
+    def _broadcast_all_clients(self, name: str, value: t.Any):
+        setattr(g, "is_broadcasting", True)
+        self._update_var(name, value)
```

Comment:
Would this also broadcast expression that depends on the var ?

Reply:
yes it will. `_update_var` will also take into account the revaluation and update the `derived_modified` list, which will propogate to all clients through `__send_ws_update_with_dict` method.